driver/main.cpp

```diff
@@ -0,0 +1,40 @@
+#include <iostream>
+#include <fstream>
+#include "reader.h"
+using namespace reader;
```

Comment:
at some point you will want to remove this and put you function in the reader namespace but thats just a reminder I bet you were already thinking that :sheep: